### DISCUSSION TITLE
I made a Spider-Man animation in ToonSquid!!

### DISCUSSION THREAD

**Discussion by CowlBlue** [keiwando/toonsquid#572] [Show and tell]
https://x.com/bluecowlboy/status/1762006499814207721

https://github.com/keiwando/toonsquid/assets/167805438/b162a500-984e-4b94-9d05-3af228a5d43b

**Comment by Cjjuntti** [comment]
Incredible! That looks like it took a long time to do. Good job!

**Reply by CowlBlue** [reply]
yes it took quite a while!

**Comment by keiwando** [comment]
Amazing!

**Reply by CowlBlue** [reply]
Thank you!!

**Comment by dualflex** [comment]
Hi BlueCowlBoy!
First of all let me say you did a great animation!
As a hobby I would like to make similar animations for my grandchilds!
Could you share some tips-books-online lessons on how you get to draw 3D figurine like that keeping relatively good proportions?
Keep up the passion!
Frank :-)

**Reply by DinoAwakening** [reply]
Hey, Frank! For you to achieve consistent proportions while drawing 3D-like bodies, in short; all you have to do is use references when needed, and understand basic shapes such as cilinders, cubes, pyramids or even spheres. It might take some time to master, nevertheless that’s the safest way to see an actual improvement :)

I hope it helps! And your art glitter your grandchildren’s eyes in excitement and pride! :D

**Reply by CowlBlue** [reply]
Hi Frank!  A very helpful tip for understanding perspective is that things close to you/ your camera are larger visually than things farther away.  If you apply this to simple 3d shapes like boxes, you’ll notice that the sides of the box will converge as the geometry is located farther away from you.  Let me know if you have any more questions!

**Comment by SharpsterDraws** [comment]
Very nice animation!

**Comment by Gottafindaname** [comment]
One question… HOW?! How did you plan for those flickering arms when he falls? Very good!
